skyvern-frontend/.env.example

```diff
@@ -8,4 +8,4 @@ VITE_ARTIFACT_API_BASE_URL=http://localhost:9090
 VITE_WSS_BASE_URL=ws://localhost:8000/api/v1
 
 # your api key - for x-api-key header
-VITE_SKYVERN_API_KEY=
\ No newline at end of file
+# VITE_SKYVERN_API_KEY=YOUR_API_KEY
```

Comment:
Can we just delete this if it's not used?

Reply:
It's not used but I thought it's there as a kind of documentation that this is a value you need to fill in for the actual `.env` file

Reply:
This is outdated now, I use it to find and replace with `sed`.